portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,29 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments()">
     <div id="content">
       <h1>My Portfolio</h1>
+      <div>
+            <h1 id="hello-container"></h1>
+      </div>
       <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Hi, welcome to my portfolio! My name is Evelyn—I'm currently
+          a rising third-year computer science major at UCSB. I love to
+          read and make music. Nice to meet you!
+      </p>
+      <img src='/images/profile.jpg' width='100px'>
+      <form action='/comment' method='POST'>
+        <p>Feel free to leave your comments here!</p>
+        <textarea name='name'>Name</textarea>
+        <br/>
+        <textarea name='comment'>Comment</textarea>
```

Comment:
You can take a look at the html attribute called "placeholder"

Reply:
I've updated the code to use the placeholder attribute rather than text